### DISCUSSION TITLE
configuration/plugins.py?

### DISCUSSION THREAD

**Discussion by etfz** [netbox-community/netbox-docker#1011] [Getting Help] [answered]
Hi,

I've only just noticed this file after noticing it was added to the `Dockerfile` example. What's the deal with this file? I don't understand exactly what the purpose of those lines in the `Dockerfile` are, but the article says to use `configuration.py` for plugin configuration, which is what I've done thus far. If I can use `plugins.py`, that seems more convenient, but it doesn't really seem to be documented.

**Comment by kkthxbye-code** [comment, marked as answer]
https://github.com/netbox-community/netbox-docker/wiki/Configuration#custom-configuration-files

Any file in the configuration folder is loaded, so it's not a special file. There is no difference between putting plugin config in configuration.py or plugins.py, or for that matter mysuperspecialcustomconfigurationfile.py.
